functions/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "owid-functions",
+  "type": "module",
```

Comment:
It declares the project is using ES modules (i.e. `import` instead of `require`). This was necessary to fix import errors after I made the `tsconfig.json` change. I think we want it in general for new code, but shouldn't matter at runtime in practice for Cloudflare functions, which only support ES modules anyway.